python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -1254,17 +1261,11 @@ def main(args):
 using namespace mlir;
 
 StringRef mlir::torch::Torch::getShapeLibrary() {{
```

Comment:
I think this is a constant string now -- you should be able to just check in ShapeLibrary.cpp and not generate it in this script.